mock/py/mockbuild/buildroot.py

```diff
@@ -207,6 +207,12 @@ def doChroot(self, command, nosync=False, *args, **kargs):
             self.uid_manager.restorePrivs()
         return result
 
+    def all_chroot_packages(self):
+        """package set, result of rpm -qa in the buildroot"""
+        out, _ = self.doChroot([self.config['rpm_command'], "-qa"],
+                               returnOutput=True, printOutput=False, shell=False)
```

Comment:
Note that this could probably also do something like util.do(rpm -qa --root slef.root) instead. Not sure what is preferred.

Reply:
will test that, it seems faster

Reply:
there are 2 things to keep in mind... we use `--nodeps` so that rpmbuild does not read rpmdb... because it is generated by outer RPM... but I don't know which command is correct for `bootstrap` chroot one.